global_config.js

```diff
@@ -0,0 +1,46 @@
+// global_config.js
+
+var Parse = require('parse/node').Parse,
+    PromiseRouter = require('./PromiseRouter');
+
+var router = new PromiseRouter();
+
+function updateGlobalConfig(req) {
+  if (!req.auth.isMaster) {
+    return Promise.resolve({
+      status: 401,
+      response: {error: 'unauthorized'},
+    });
+  }
+
+  return req.config.database.rawCollection('_GlobalConfig')
+    .then(coll => coll.findOneAndUpdate({ _id: 1 }, { $set: req.body }, { returnOriginal: false }))
+    .then(response => {
+      return { response: { params: response.value.params } }
+    })
+    .catch(() => ({
+      status: 404,
+      response: {
+        code: 103,
```

Comment:
Error 103 is for invalid class names. I think you probably want `Parse.Error.INVALID_KEY_NAME` maybe. There could be a variety of things that cause this request to fail. Maybe we should return specific errors depending on what exactly went wrong.